app/models/job.rb

```diff
@@ -55,11 +55,22 @@ def pct_format(number)
 
     def app_inspect(bin_count = 5, job_name = 'ondemand/sys/dashboard/sys/bc_osc_jupyter', cluster = '_all', property = 'CPUs')
       property_hash = Hash.new(0)
-      match_str = property == 'GPUs' ? /gpu:[^,]*(\d+)/ : /cpu=(\d*)/
-      Job.all.each do |job|
-        next unless job.name == job_name && job.tres && (cluster == '_all' || job.cluster == cluster)
+      if property == 'Hours'
+        Job.all.each do |job|
+          next unless job.name == job_name && (cluster == '_all' || job.cluster == cluster)
+
+          s = job.elapsed.split('-')
+          # hours = hours + (days * 24 if days are included)
+          hours = s[-1].split(':')[0].to_i + (s.length == 2 ? 24 * s[0].to_i : 0)
```

Comment:
Shame we have to parse this ourselves, but that seems to be the case. I can't see an easy way to to use `ActiveSupport::Duration` that's any nicer than this.